.gitignore

```diff
@@ -1,12 +1,7 @@
 *.iml
-.gradle
+.gradle/*.*
+.idea/*.*
```

Comment:
There is now a lot of `.idea` files committed in this PR. This is probably not working